core/src/main/java/tc/oc/pgm/core/CoreModule.java

```diff
@@ -118,13 +119,14 @@ public CoreModule parse(MapFactory context, Logger logger, Document doc)
         }
 
         String modes = coreEl.getAttributeValue("modes");
-        List<String> modeList = new ArrayList<>();
+        List<Mode> modeList = new ArrayList<>();
         if (modes != null) {
-          Node node = Node.fromAttr(coreEl, modes);
-          if (node != null) {
-            for (String mode : Splitter.on(" ").split(node.getValue())) {
-              modeList.add(mode);
-            }
+          for (String mode : Splitter.on(" ").split(modes)) {
```

Comment:
```suggestion
          for (String mode : Splitter.on("\\s").split(modes)) {
```
use whitespace as a separator, similar to the example in https://github.com/PGMDev/PGM/blob/dev/core/src/main/java/tc/oc/pgm/flag/FlagParser.java#L105